PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -283,11 +300,18 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         cms.PSet( name = cms.string("LepGood_sip3d"), expr = cms.string("abs(dB('PV3D')/edB('PV3D'))")),
         cms.PSet( name = cms.string("LepGood_dz"), expr = cms.string("log(abs(dB('PVDZ')))")),
         cms.PSet( name = cms.string("LepGood_mvaFall17V2noIso"), expr = cms.string("userFloat('mvaNoIso_Fall17V2')")),
-    )
 )
+
 run2_egamma_2016.toModify(
     electronPROMPTMVA,
     weightFile = "PhysicsTools/NanoAOD/data/el_BDTG_2016.weights.xml",
+    variables = _legacy_electron_BDT_variable
+)
+
+(run2_egamma_2016 | run2_egamma_2018).toModify(
```

Comment:
@Cvico I think it should be `run2_egamma_2017` instead of `run2_egamma_2016`?

Reply:
Hi @hqucms. You are right, I've just pushed the fix. Thanks!